src/pages/programs/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { useState } from "react";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import Modal from "../../components/grid-aware/Modal";
+import ProgramBlock from "../../components/grid-aware/ProgramBlock";
+import ProgramsPageLargeParagraph from "../../components/grid-aware/ProgramsPageLargeParagraph";
+import ProgramsSubTitleBlock from "../../components/grid-aware/ProgramsSubTitleBlock";
+import Spacer from "../../components/grid-aware/Spacer";
+import Layout from "../../components/layout";
+import PartnershipSignupForm from "../../components/thirdparty/mailchimp/PartnershipSignupForm";
+import communityDevelopmentImg from "./community-development.jpg";
+import articleSpotlightImage from "./laura-barerra-vera.png";
+import serviceGuideImg from "./sf-service-guide.jpg";
+import shelterConnect from "./shelter-connect.jpg";
+
+export default () => {
+  const [partnershipFormIsOpen, setPartnershipFormIsOpen] = useState(false);
+
+  return (
+    <Layout>
+      <Modal
+        isOpen={partnershipFormIsOpen}
+        setIsOpen={setPartnershipFormIsOpen}
+        contentLabel="Partnership"
+      >
+        <PartnershipSignupForm />
+      </Modal>
+      <ProgramsPageLargeParagraph
+        title="Our Programs"
+        description="ShelterTech's programs are designed to suppport the underserved community experiencing homelessness and housing insecurity through WiFi connectivity and acccess to online resources."
+      />
+      <ProgramsSubTitleBlock title="ShelterTech offers the following programs:" />
+      <ProgramBlock
+        title="SF Service Guide"
+        subtitle="an online directory of human services in San Francisco"
+        description={
+          <>
+            <p>
+              The SF Service Guide is an online directory of human services in
+              San Francisco that launched in November 2019 and currently serves
+              1,500 unique monthly users. The directory&apos;s focus is on
+              homelessness and housing services but also covers a variety of
+              other services, from education and legal aid to senior services
+              and re-entry programs. Our goal is to help anyone with access to
+              smartphone, tablet, or computer find services they need.
+            </p>
+            <p>
+              SF Service Guide is supported by a grant from the SF Mayor&apos;s
+              Office of Housing and Community Development and verified for
+              accuracy by our data partner, the Homeless Advocacy Project of the
+              JDC.
+            </p>
+          </>
+        }
+        image={{
+          url: serviceGuideImg,
+          alt: "SF Service Guide",
+        }}
+        ctaButtons={[
+          {
+            text: "Visit Site",
+            externalLink: "https://sfserviceguide.org/",
+          },
+          {
+            text: "Donate",
+            externalLink: "https://www.sheltertech.org/donate/",
```

Comment:
For this and all the other CTA links to the Donate page, could you use an internalLink instead of an externalLink? Making it an external link causes it to open in a new tab instead of directly taking you there.